serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -33,6 +33,19 @@ Follow the <<building-kubernetes, Kubernetes>> or <<building-openshift, OpenShif
 
 The operator can build workflows on Kubernetes or OpenShift. On Kubernetes, it uses link:{kaniko_url}[Kaniko] and on OpenShift a link:{openshift_build_url}[standard BuildConfig]. The operator build system is not tailored for advanced use cases and you can do only a few customizations.
 
+=== Using another Workflow base builder image
+
+If your scenario has strict policies for image usage, such as security or hardening constraints, you can replace the default image used by the operator to build the final workflow container image. Alternatively, you might want to test a nightly build with a bug fix or a custom image containing your customizations.
+
+By default, the operator will use the image distributed upstream to run the workflows in the development profile. You can change this image by editing the `SonataFlowPlatform` custom resource in the namespace where you deployed your workflows:
+
+.Patching the current SonataFlowPlatform with the new builder image
+[source,bash,subs="attributes+"]
+----
+# use `kubectl get sonataflowplatform` to get the SonataFlowPlatform name
+kubectl patch sonataflowplatform <name> --patch  'spec:\n plaform:\n baseImage: <your new image full name with tag>' -n <your_namespace>
+----
+
 === Changing the base build Dockerfile
```

Comment:
@ricardozanini Would we like to remove this section or is it better to leave it since at the moment we haven't got another way to customize the Dockerfile?

Reply:
We can keep this section, and rename it to "Customize the base build Dockerfile", I think it's better. We can get rid of the "base name" explanations.